app/templates/components/forms/wizard/sponsors-step.hbs

```diff
@@ -66,7 +66,11 @@
         <div class="ui section divider"></div>
       {{/if}}
     {{/each}}
-    <button type="button" class="ui primary {{if device.isMobile 'small'}} button" {{action 'addSponsor'}}>{{t 'Add another sponsor'}}</button>
+    {{#if (eq sponsors.length 0)}}
```

Comment:
We may just use 
```hbs
{{#if sponsors.length}}
```
Can you check? @Anupam-dagar

Reply:
Update PR with the changes.